app/templates/gentelella/users/events/_table.html

```diff
@@ -72,15 +72,15 @@
                         {% if flag %}
                             {% for id, item in ticket_stats.iteritems() %}
                                 <div>
-                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span> 
+                                    <span>{{ item.name }}</span> <span> ({{item.completed}}/{{item.total}}) </span>
                                 </div>
                                 <br>
                              {% endfor %}
                         {% else %}
                             {{ _("No Ticket Information") }}
                         {% endif %}
                     </td>
-                    <td><a href="{{ url }}" target="_blank">{{ url }}</a></td>
+                    <td style="width:200px;"><a href="{{ url }}" target="_blank">{{ url }}</a></td>
```

Comment:
@niranjan94  I have checked .  In our project  width of the columns are given by 
table td[class*=col-],table th[class*=col-]   individual column width is not specified. 

what can be done is 

td:last-child{
    width:200px;
}
because in the issue it's saying just to change the width of the last column.